tac/project-updates/2025/2025-annual-Cacti.md

```diff
@@ -0,0 +1,90 @@
+</mark>
+
+- We have **8 maintainers** at present from 4 different organizations: Fujitsu, IBM, Tecnico Lisboa, Accenture.
+- The number of active maintainers has **increased** from 7 to 8.
+- The number of maintaining organizations has **increased** from 3 to 4.
+
+Current [MAINTAINERS.md](https://github.com/hyperledger-cacti/cacti/blob/36959b326f8a4c08c99fe2bf87b00c7996d49edd/MAINTAINERS.md)
+and previous (Jan 1st, 2024) [MAINTAINERS.md](https://github.com/hyperledger-cacti/cacti/blob/1fb2551a055daf202d2def0fcb3de990a44cfb03/MAINTAINERS.md)
+
+## Project Adoption
+
+The detailed answer about adoption has been published earlier and can be read through this link: https://github.com/hyperledger/hyperledger-hip/blob/gh-pages/HIPs/graduated/cacti.md#real-world-use
+
+## Goals
+
+### Performance Against Prior Goals
+
+The main theme is to carry on with the integration work with our multi-phase plan:
+
+> Packaging as a monorepo,
+
+Great success here! With the introduction of our new bundler (`npm:@vercel/ncc`) we now have the capability to bundle
+any package within the monorepo that has application characteristics (e.g. anything that's runnable instead of designed
+to be imported as a library only) along with it's mono-repo dependencies into a single executable file. This helps
+tremendously with container sizes and container build times for the `cmd-api-server` package for example and the IETF SATP
+gateway package recently started using it as well.
+
+> Eliminating and merging redundant packages,
+> Finally consolidating packages as per our target architecture (this may be too ambitious for 2024, given other priorities in the roadmap).
+
+Semi-success. We have a long way to with this task still, but much progress has been achieved.
+Our excellent mentee Jennifer has done an outstanding job of bringing it all together through the new COPM module which 
+was also recently presented at an LFDT meetup.
+This package increases re-use by enabling certain Weaver components to be used from other non-weaver packages.
+
+> Adding more plugins for other forms of DLTs and also legacy/centralized systems that need to trade assets.
+
+Great success! The daily pair programming sessions shined on this one! Peter and Cheesecake Labs' Fifo have worked together on
+the Stellar connector which was then also presented at a conference by Cheesecake Labs.
+https://github.com/hyperledger-cacti/cacti/pull/3296
+https://github.com/hyperledger-cacti/cacti/pull/3274
+
+> Issuance of the much anticipated v2.0.0 final/GA/etc. release in Q1 or Q2 the latest!
+
+We've successfully released v2.0.0 and v2.1.0 in the meantime. 
+Artifact publishing is on hold at the moment due to the migration effort from the old org to the new org.
+
+
+### Next Year's Goals
+
+1. Have the Typescript reference implementation of the IETF Secure Asset Transfer Protocol (SATP) deployed to at least one production project.
```

Comment:
`deployed to at least one production project.` I struggle with this goal as it's hard to measure this from an open source perspective. Would be interested to redefine this goal as maybe `Helping production project with a dedicated discord channel and support`

Reply:
@EnriqueL8 Got it, let me refactor.

Reply:
@EnriqueL8 https://github.com/LF-Decentralized-Trust/governance/pull/108/commits/bd85b1a3cf7fdd2d0f655e9a161abebaf9ad209c